engine/src/xetex_math.rs

```diff
@@ -1975,7 +1979,7 @@ pub unsafe extern "C" fn after_math() {
         .b32
         .s1 as isize,
     ) < 22i32
-        && !(*font_area.offset(
+        && !(font_area[
             (*eqtb.offset(
```

Comment:
Magic numbers that can be replaced. See https://github.com/tectonic-typesetting/tectonic/blob/master/tectonic/xetex-math.c